data/recognizer_registry/QUT/data.yml

```diff
@@ -778,32 +778,73 @@
         tn: 0
         fn: 3616
 
-  repository: https://github.com/QutEcoacoustics/audio-analysis/blob/master/src/AnalysisConfigFiles/RecognizerConfigFiles/Truskinger.ManorinaMelanophrys.yml
+
+- title: Kookaburra Recognizer
+  description: Recognizer for the Laughing Kookaburra (Dacelo novaeguineae)
+  targets:
+    - name: Kookaburra
+      binomial: Dacelo
+      variants:
+
+  repository: https://github.com/QutEcoacoustics/recognizer_workshop/tree/main/artifacts/models/kookaburra
   organization: QUT Ecoacoustics
   # https://citation-file-format.github.io/
   # cff-version: 1.2.0
   message: "If you use this software, please cite it as below."
   authors:
-    - familyNames: Towsey
-      givenNames: Michael
-      orcid: 0000-0002-8246-7151
-    - familyNames: Truskinger
-      givenNames: Anthony
-      orcid: 0000-0003-1110-1483
-    - familyNames: Thompson
-      givenNames: Kristen
+    - familyNames: De Vine
+      givenNames: Lance
       orcid:
-  version: 1.0.0
-  doi: 10.5281/zenodo.1188743
-  dateReleased: 2020-12-16
-  license: Apache-2.0
-  
-  
-  - title: Kookaburra Recognizer
-  description: 
+  version: 0.0.1
+  doi: 10.5281/zenodo.8045883
+  dateReleased: 2023-06-16
+  license: Creative Commons Attribution-ShareAlike 4.0 International
+
+- title: Noisy Pitta Recognizer
+  description: Recognizer for the Noisy Pitta (Pitta versicolor)
   targets:
-    - name: Kookaburra
-      binomial: Dacelo
+    - name: Noisy Pitta
+      binomial: Pitta versicolor
       variants:
+  datasets:
+    training:
+      summary: 46 positive, 65 negative
```

Comment:
this is an incorrect structure: https://github.com/ecoacoustics/website/blob/8de004713958a333a12a72b34834077e91189e51/static/registry_schema.json#L133-L161